README.md

```diff
@@ -174,33 +192,31 @@ Spawns the following Files:
 
 
 
+
 ## Project Structure
 There is a `public` folder for static files and a `src` folder for the fancy app code.
 
-Inside the `src` folder three main sections can be found. Components are located 
-in the `components` folder each component has its own folder which contains all resources
-used by the component.
-
-The `shared` folder is for code which is shared and can be consumed from many components and other
-parts of the app. Like utility functions, services etc.
-
-As Cerebral is used for state and side effects management the `modules` folder is introduced 
+As CerebralJS is used for state and side effects management the `modules` folder is introduced 
 to split Cerebral's state into modules. Therefore each module has its own folder 
 which contains signals, actions, computes, and so on.
 
 ```
 |-public
     |-index.html
-    |-manifest.json
 |-src
     |-components 
+        |-Button
+            |-Button.js
+            |-Button.md
+            |-Button.module.js
```

Comment:
shouldn't this be Button.module.scss

Reply:
It should! Thanks for pointing that out.